### ISSUE TITLE
How to integrate png or svg in javadoc html

### ISSUE THREAD

**Issue by kst83** [talsma-ict/umldoclet#61]
Is there a way to integrate the resulting diagrams (png or svg) in the html generated?

**Comment by sjoerdtalsma** [comment]
For the moment, you can include the image by adding a regular HTML `<img>` tag in your JavaDoc. The URL can be relative to the html, or you can define an `imagedir` to be used.

For the `2.0` version, we're looking at options for automatic inclusion in the generated HTML documentation.

**Comment by sjoerdtalsma** [comment]
Before the final `2.0.0` release, we should at least look into the feasibility of providing an automated feature for this.

**Comment by sjoerdtalsma** [comment]
I think a feasible strategy is to implement a custom [Taglet](https://docs.oracle.com/javase/9/docs/api/jdk/javadoc/doclet/Taglet.html) and provide it to the Standard doclet when generating the HTML

**Comment by sjoerdtalsma** [comment]
:disappointed:  A taglet cannot be added from the same path specification as the doclet.
This requires either a different Jar purely for adding the UML image to the HTML _or_ adding the doclet jar to _both_ the `docletPath` and the `tagletPath` when running Javadoc.

Maybe a post-processing phase to manipulate the HTML isn't such a bad idea after all; this is the approach followed by others as well.

**Comment by sjoerdtalsma** [comment]
Cursory investigation shows that for generated Javadoc class HTML, simply replacing the first
```html
<hr>
```
by
```html
<hr>
<img src="MyClass.svg" alt="MyClass class diagram" style="float: right;">
```
may turn out quite satisfactory...

**Comment by sjoerdtalsma** [comment]
Now also available on JDK 1.7 + 8: use UML doclet version `1.1.0`
